lib/Config.ts

```diff
@@ -59,6 +59,8 @@ class Config {
 
     this.p2p = {
       listen: true,
+      discover: false,
```

Comment:
Default should be `true` I think, I think the typical user would want to connect to peers automatically.

Reply:
Oh okay, I thought you wanted vice versa.

Reply:
@kilrau ?

Reply:
This was discussed this on the weekly call last week I believe. The default behavior we want is a fully connected network.